website/documentation/guides/security-and-compliance/kubernetes-hardening/_index.md

```diff
@@ -0,0 +1,118 @@
+|242388|The Kubernetes API server must have the insecure bind address not set.|||
+|242389|The Kubernetes API server must have the secure port set.|||
+|242391|The Kubernetes Kubelet must have anonymous authentication disabled.|||
+|242392|The Kubernetes kubelet must enable explicit authorization.|||
+|242396|Kubernetes Kubectl cp command must give expected access and results.|||
+|242397|The Kubernetes kubelet staticPodPath must not enable static pods.|||
+|242404|Kubernetes Kubelet must deny hostname override.|||
+|242405|The Kubernetes manifests must be owned by root.|||
+|242406|The Kubernetes KubeletConfiguration file must be owned by root.|||
+|242407|The Kubernetes KubeletConfiguration files must have file permissions set to 644 or more restrictive.|||
+|242408|The Kubernetes manifest files must have least privileges.|||
+|242409|Kubernetes Controller Manager must disable profiling.|||
+|242410|The Kubernetes API Server must enforce ports, protocols, and services (PPS) that adhere to the Ports, Protocols, and Services Management Category Assurance List (PPSM CAL).|||
+|242411|The Kubernetes Scheduler must enforce ports, protocols, and services (PPS) that adhere to the Ports, Protocols, and Services Management Category Assurance List (PPSM CAL).|||
+|242412|The Kubernetes Controllers must enforce ports, protocols, and services (PPS) that adhere to the Ports, Protocols, and Services Management Category Assurance List (PPSM CAL).|||
+|242413|The Kubernetes etcd must enforce ports, protocols, and services (PPS) that adhere to the Ports, Protocols, and Services Management Category Assurance List (PPSM CAL).|||
+|242418|The Kubernetes API server must use approved cipher suites.|||
+|242419|Kubernetes API Server must have the SSL Certificate Authority set.|||
+|242420|Kubernetes Kubelet must have the SSL Certificate Authority set.|||
+|242421|Kubernetes Controller Manager must have the SSL Certificate Authority set.|||
+|242422|Kubernetes API Server must have a certificate for communication.|||
+|242423|Kubernetes etcd must enable client authentication to secure service.|||
+|242424|Kubernetes Kubelet must enable tlsPrivateKeyFile for client authentication to secure service.|||
+|242425|Kubernetes Kubelet must enable tlsCertFile for client authentication to secure service.|||
+|242426|Kubernetes etcd must enable client authentication to secure service.|||
+|242427|Kubernetes etcd must have a key file for secure communication.|||
+|242428|Kubernetes etcd must have a certificate for communication.|||
+|242429|Kubernetes etcd must have the SSL Certificate Authority set.|||
+|242430|Kubernetes etcd must have a certificate for communication.|||
+|242431|Kubernetes etcd must have a key file for secure communication.|||
+|242432|Kubernetes etcd must have peer-cert-file set for secure communication.|||
+|242433|Kubernetes etcd must have a peer-key-file set for secure communication.|||
+|242434|Kubernetes Kubelet must enable kernel protection.|||
```

Comment:
This rule is already mentioned in the "Control Plane Configuration" section so we can remove it from here.

```suggestion
```